frontend/src/APIClients/LogRecordAPIClient.ts

```diff
@@ -85,28 +87,83 @@ const filterLogRecords = async ({
   }
 };
 
-const createLog = async (
-  userId: number,
-  residentId: number,
-  flagged: boolean,
-  note: string,
-  attentionTo: number,
-  building: string,
-): Promise<PostLogRecordsResponse> => {
+const createLog = async ({
+  employeeId,
+  residentId,
+  datetime,
+  flagged,
+  note,
+  tags,
+  building,
+  attnTo,
+}: CreateLogRecordParams): Promise<PostLogRecordsResponse> => {
   try {
     const bearerToken = `Bearer ${getLocalStorageObjProperty(
       AUTHENTICATED_USER_KEY,
       "accessToken",
     )}`;
-
     const { data } = await baseAPIClient.post<PostLogRecordsResponse>(
       "/log_records/",
       {
-        employeeId: userId,
+        employeeId,
+        residentId,
+        datetime,
+        flagged,
+        note,
+        tags,
+        building,
+        attnTo,
+      },
+      { headers: { Authorization: bearerToken } },
+    );
+    return data;
+  } catch (error) {
+    return null;
+  }
+};
+
+const deleteLogRecord = async (logId: number): Promise<boolean> => {
+  try {
+    const bearerToken = `Bearer ${getLocalStorageObjProperty(
+      AUTHENTICATED_USER_KEY,
+      "accessToken",
+    )}`;
+    await baseAPIClient.delete(`/log_records/${logId}`, {
+      headers: { Authorization: bearerToken },
+    });
+    return true;
+  } catch (error) {
+    return false;
+  }
+};
+
+const editLogRecord = async ({
+  logId,
+  employeeId,
+  residentId,
+  datetime,
+  flagged,
+  note,
+  tags,
+  building,
+  attnTo,
+}: EditLogRecordParams): Promise<any> => {
```

Comment:
Could you make a return type for this

Reply:
Just adding to this, this is what the endpoint returns:

![Screen Shot 2023-07-29 at 11 34 42 PM](https://github.com/uwblueprint/supportive-housing/assets/53386475/889ea50e-b5a3-41f9-bdb1-7583b47ec8b9)

Thus I think it suffices to just have boolean as the type

```suggestion
}: EditLogRecordParams): Promise<boolean> => {
```

Reply:
It'd also make it consistent with what Daniel implemented [in his PR](https://github.com/uwblueprint/supportive-housing/pull/143/files#diff-8e91a0f35a9a0a0ae5ced12438049485b3f15186ace43fe8d009acd597ece22aR107)